src/datapump/jobs/geotrellis.py

```diff
@@ -0,0 +1,387 @@
+from typing import Dict, Any, List
+from itertools import groupby
+from pathlib import Path
+from pprint import pformat
+from enum import Enum
+
+from ..globals import (
+    S3_BUCKET_PIPELINE,
+    EC2_KEY_NAME,
+    PUBLIC_SUBNET_IDS,
+    LOGGER,
+    EMR_VERSION,
+    GEOTRELLIS_JAR_PATH,
+    WORKER_COUNT_MIN,
+    WORKER_COUNT_PER_GB_FEATURES,
+    ENV,
+)
+from ..clients.aws import get_emr_client, get_s3_client, get_s3_path_parts
+from ..commands import Analysis, AnalysisInputTable
+from ..jobs.jobs import (
+    AnalysisResultTable,
+    Job,
+    JobStatus,
+)
+
+WORKER_INSTANCE_TYPES = ["r4.2xlarge", "r5.2xlarge"]
+MASTER_INSTANCE_TYPE = "r4.2xlarge"
+
+
+class GeotrellisAnalysis(str, Enum):
+    """
+    Supported analyses to run on datasets
+    """
+
+    tcl = "annualupdate_minimal"
+    glad = "gladalerts"
+    viirs = "firealerts_viirs"
+    modis = "firealerts_modis"
```

Comment:
I guess the difference is that this is specifically mapping the names of these analyses we use in geotrellis, vs just the general names I'm referring to. I.e. if we start using a different library to do the same analysis, the other enum would stay the same, but this may change. Or there can be analyses in the other enum that aren't using geotrellis.